pkgs/applications/audio/pd-plugins/gem/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, autoreconfHook,
+  autoconf, automake, libtool, pkgconfig,
+  puredata,
+  libGL, libGLU, glew, freeglut, libv4l, libX11,
+  file
+ }:
+
+stdenv.mkDerivation rec {
+  pname = "gem";
+  version = "unstable-2020-03-26";
+
+  src = fetchFromGitHub {
+    owner = "umlaeute";
+    repo = "Gem";
+    rev = "f38748d71bfca00e4d2b2f31d6c4e3759c03d599";
+    sha256 = "0bkky5fk0a836bapslrgzil272iq9y704y7hw254cfq5ffjd4qjy";
+  };
+
+  # https://github.com/umlaeute/Gem/pull/251
+  patches = [
+    ./0001-Update-discovery-of-OpenGL-GLU-GLUT-libraries.patch
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    autoconf
```

Comment:
You should never need both `autoreconfHook` _and_ `autoconf` + `automake`. The 1st includes the 2 others.